WootricSDK/WootricSDK/WTRScoreView.m

```diff
@@ -29,13 +29,19 @@
 @interface WTRScoreView ()
 
 @property (nonatomic, assign) int currentValue;
+@property (nonatomic, strong) UIColor *labelColor;
 
 @end
 
 @implementation WTRScoreView
 
 - (instancetype)init {
+  return [self initWithColor:[WTRColor selectedValueScoreColor]];
+}
+
+- (instancetype)initWithColor:(UIColor *)color{
```

Comment:
this is just optional (it's because my OCD), there's a missing space between _color_ and _{_